main.scss

```diff
@@ -23,19 +23,16 @@
 /// @require {mixin} oOverlay
 /// @require {mixin} oOverlayShadow
 /// @require {mixin} oOverlayArrow
```

Comment:
You can get rid all require declarations, since the new version of SassDoc now computes dependencies properly.